build_projects/dotnet-host-build/PublishTargets.cs

```diff
@@ -288,6 +298,13 @@ private static bool CheckIfAllBuildsHavePublished(BuildTargetContext c)
         public static BuildTargetResult PublishArtifacts(BuildTargetContext c) => c.Success();
 
         [Target(
+            nameof(PublishTargets.PublishInstallerFilesToAzure),
+            nameof(PublishTargets.PublishDotnetDebToolPackage),
+            nameof(PublishTargets.PublishDebFilesToDebianRepo),
+            nameof(PublishTargets.PublishSharedFrameworkVersionBadge))]
```

Comment:
One can use the Environment variable check to determine a target to be called or not.  I have followed this approach of grouping the applicable targets.